app/pathfinder_gui.py

```diff
@@ -238,11 +238,14 @@ def generate_links(path):
             'vulnerabilityreports', match=dict(id=report_id)
         )
         tags = data.pop('adversary_tags')
+        whitelist = [] #data.pop('whitelist')
```

Comment:
We may consider shifting the language to `allow = []` and `deny = []`.  This change would also line these lists up with the `Enum` for hosts status.